IronFoundry.Container/ContainerHostDependencyHelper.cs

```diff
@@ -47,8 +47,9 @@ IEnumerable<string> EnumerateLocalReferences(Assembly assembly)
                 {
                     yield return referencedAssembly.Location;
 
-                    foreach (var nestedReferenceFilePath in EnumerateLocalReferences(referencedAssembly))
-                        yield return nestedReferenceFilePath;
+                    if (!referencedAssembly.Location.Contains("ICSharpCode.SharpZipLib.dll"))
```

Comment:
I was hoping we fixed this issue by correctly filtering out assemblies that are in the GAC.  Is this issue still occurring in the latest IronFoundry.Container library?